scripts/wail.coffee

```diff
@@ -14,14 +14,22 @@
 #   csoni111
 
 module.exports = (robot) ->
-  robot.respond /who.*lab/i, (msg) ->
-    waliUrl = process.env.WAIL_PIC_URL + '?t=' + new Date().getTime()
-    msg.send(
-      "attachments": [
+    robot.respond /who.*lab/i, (msg) ->
+        waliUrl = process.env.WAIL_PIC_URL + '?t=' + new Date().getTime()
+        wailTextUrl = process.env.WAIL_NAME_URL + '?t=' + new Date().getTime()
+        
+        request = require 'request'
```

Comment:
There's no `request` package defined in dependencies in `package.json`. Instead you could use `robot.http()` to make a http call. Check [here](https://github.com/hubotio/hubot/blob/master/docs/scripting.md#making-http-calls).